src/lxc/tools/lxc_ls.c

```diff
@@ -1290,3 +1323,51 @@ static void ls_field_width(const struct ls *l, const size_t size,
 		}
 	}
 }
+
+static int rm_r(char *dirname)
```

Comment:
Hm, you could also move `static int recursive_destroy(char *dirname)` from `cgroups/cgfsng.c` to `utils.{c,h}` and call it `rm_r()` and use it in both places.

Reply:
I'll check it you mentioned after tools are merged.